bigquery/google/cloud/bigquery/_helpers.py

```diff
@@ -472,13 +474,35 @@ class QueryParametersProperty(object):
     """Custom property type, holding query parameter instances."""
 
     def __get__(self, instance, owner):
-        """Descriptor protocol:  accessor"""
+        """Descriptor protocol:  accessor
+
+        :type instance: :class:`QueryParametersProperty`
+        :param instance: instance owning the property (None if accessed via
+                         the class).
+
+        :type owner: type
+        :param owner: the class owning the property.
+
+        :rtype: list of instances of classes derived from
+                :class:`AbstractQueryParameter`.
+        :returns: the descriptor, if accessed via the class, or the instance's
+                  query paramters.
+        """
         if instance is None:
             return self
         return list(instance._query_parameters)
 
     def __set__(self, instance, value):
-        """Descriptor protocol:  mutator"""
+        """Descriptor protocol:  mutator
+
+        :type instance: :class:`QueryParametersProperty`
+        :param instance: instance owning the property (None if accessed via
+                         the class).
+
+        :type value: list of instances of classes derived from
+                :class:`AbstractQueryParameter`.
```

Comment:
I pushed the indentation change and cancelled the Travis / Circle builds.  Any remaining issues before squashing merging?